core/amber/src/main/scala/edu/uci/ics/texera/web/OPversion.java

```diff
@@ -14,7 +14,7 @@ public class OPversion {
     static {
         try {
             Path path = Paths.get(currentPath);
-            if(currentPath.contains("core/amber")){
+            if(path.endsWith("core/amber/")){
```

Comment:
i suggest we also convert the string "core/amber/" to a path object, `Paths.get("core/amber/")`